src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -495,9 +500,14 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   /** Model value for "same as (records) mailing address" checkbox. */
   private inheritRecMailingAddress = true
 
-  /** Is true when the mailing address is not in BC, Canada. */
-  get disableSameDeliveryAddress (): boolean {
-    return (
+  /**
+   * Sets the "disableSameDeliveryAddress" flag.
+   * Needed because "this.mailingAddress" doesn't seem to be reactive.
+   */
```

Comment:
Yes, this is a workaround. I don't know why `this.mailingAddress` is not reactive. It's correctly initialized as `{}` (ie, not undefined) so 🤷‍♂️

Reply:
After thinking about this, I realized that `this.mailingAddress` **is** reactive, but when there is an updated address, we use Object.assign() to update its properties. Since getters don't see "deep" changes (ie, to object properties), only object variable properties, the getter (in the old code) didn't realize that the address had been updated and it provided its old (cached) value.

If, instead, we reassigned `this.mailingAddress`, then the reactive functions would execute and the getter would indeed provide an updated value. I didn't do this initially because it caused an infinite update loop.

I think I have a solution to the infinite loop issue, so I will rework my code ASAP.

Reply:
Obsolete comment. I'm resolving it but not deleting it in case anyone wants to learn a bit about this. See also [this comment](https://github.com/bcgov/business-edit-ui/pull/292#discussion_r893690261) on the topic.